### PR TITLE
added coordinator balance to health endpoint

### PR DESCRIPTION
**Pull request by Mikelle**
Closes https://github.com/hermeznetwork/hermez-node/issues/859

### What does this PR does?

Added coordinator balance to health endpoint

### How to test?

Run request to /v1/health to a node in coord mode.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Include tests
- [x] Respect code style and lint
- [ ] Update swagger file (if needed)
- [ ] Update documentation (*.md) (if needed)

### CODE REVIEW

File: api/swagger.yml [hunk 1]
```diff
@@ -3457,6 +3457,10 @@ components:
           type: string
           description: Current version of the hermez-node
           example: v1.2.0
+        coordinatorBalance:
```
Comment (arnaubennassar) [inline api/swagger.yml:3460]: Coordinators can use two addresses: one for forging and one for bidding. Most of the cases this address is the same, but not all the times. I'd suggest to:

a) Specify in the swagger doc that this address refers to the one used for forging (and probably change the name of `coordinatorBalance`) to help users understand
b) Add the bidder address as well

**Review by arnaubennassar — DISMISSED** [review]
Approving, as it's good enough. But please, consider implementing one of the two suggestion I've made

**Comment by arnaubennassar** [comment]
Also, could you add tests for the endpoint?

**Comment by Mikelle** [comment]
> Also, could you add tests for the endpoint?

Unfortunately, to test this field, we have to run node in coord mode and specify coordinator forger address. 
Attaching screenshot from such node, to see, that coordinatorForgerAddress is returning.
Screen Shot 2021-06-25 at 1 48 10 PM

**Review by mfcastellani — APPROVED** [review]

**Review by arnaubennassar — APPROVED** [review]

**Pull request merged**